requirements/image.txt

```diff
@@ -2,3 +2,4 @@ scipy
 torchvision  # this is needed to internally set TV version according installed PT
 torch-fidelity
 lpips
+Pillow==8.4.0
```

Comment:
only one version or could it be some tide range?

Reply:
This is the latest release need for it to work. Do older packages may also

Reply:
how much older?